packages/gestalt/src/Tabs.css

```diff
@@ -29,9 +29,14 @@
 
 .tabIsNotActive:focus,
 .tabIsNotActive:hover {
+  background-color: #f1f1f1;
   color: #111;
 }
 
+.tabIsNotActive:active {
+  background-color: #e8e8e8;
```

Comment:
I see this and #f1f1f1 are also used in `Button.css`. Maybe they should be named colors in `Colors.scss` for sharing?